commands/help-text_test.go

```diff
@@ -0,0 +1,80 @@
+package commands
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+//New commands must be added here
+var helpText = map[string]string {
+	"git-lfs-checkout.1.ronn":git_lfs_checkout_HelpText,
```

Comment:
Go doesn't really have a way to reflect over variables on a class, so I settled for a static map. If anyone has better ideas on how to map the man file names to variable names I'm all ears.